pkgs/development/libraries/tbb/default.nix

```diff
@@ -101,6 +63,6 @@ stdenv.mkDerivation rec {
       details and threading mechanisms for scalability and performance.
     '';
     platforms = platforms.unix;
-    maintainers = with maintainers; [ thoughtpolice ];
+    maintainers = with maintainers; [ thoughtpolice tmarkus ];
```

Comment:
Silly question: why it's `tmarkus` and not you?

Reply:
@hesiod initially started this effort (https://github.com/NixOS/nixpkgs/pull/215689) and i wanted to help to fix the build and it escalated at bit. At this point i might be qualified to maintain it, but i already maintain too many packages!